python/tvm/relay/op/tensor.py

```diff
@@ -397,6 +397,24 @@ def divide(lhs, rhs):
     return _make.divide(lhs, rhs)
 
 
+def floor_divide(lhs, rhs):
```

Comment:
Add the op into api docs, docs/api/python/topi.rst, doc/langref/relay_op.rst, docs/frontend/tensorflow.rst